README.md

```diff
@@ -106,6 +106,13 @@ Using wicked_pdf_helpers with asset pipeline raises `Asset names passed to helpe
 </html>
 ```
 
+### Asset Pipeline usage
+
+It's best to serve asset files from the Asset Pipeline. The correct way to handle this is to include these assets in your ```config/initializers/assets.rb```.
```

Comment:
I do think this is an improvement, but this phrase is ambiguous. How about something more like:

"It's best to precompile assets used in PDF views. This will help avoid issues when it comes to deploying, as Rails serves asset files differently between development and production (`config.assets.comple = false`), which can make it look like your PDFs work in development, but fail to load assets in production."

I'm not sure I love that either. Seems a little long and overly technical. Any suggestions would be appreciated.